tools/check-commits.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+master_branch="origin/master"
+current_branch="$(git rev-parse --abbrev-ref HEAD)"
+
+revs=$(git rev-list "${master_branch}".."${current_branch}")
+
+for commit in ${revs};
```

Comment:
do we need all the commits to be with the ticket? or at least one?

Reply:
Each commit